Kickstarter-iOS/Views/Cells/ProjectPamphletMainCell.swift

```diff
@@ -234,6 +236,18 @@ internal final class ProjectPamphletMainCell: UITableViewCell, ValueCell {
         self?.delegate?.projectPamphletMainCell(_self, goToCreatorForProject: $0)
     }
 
+    self.viewModel.outputs.opacityForViews
+      .observeForUI()
+      .observeValues { [weak self] alpha in
+        guard let _self = self else { return }
+        UIView.animate(withDuration: (alpha == 0.0 ? 0.0 : 0.3), delay: 0.0, options: .curveEaseOut,
+                       animations: {
+                        _self.creatorStackView.alpha = alpha
+                        _self.statsStackView.alpha = alpha
+                        _self.blurbAndReadMoreStackView.alpha = alpha
+        }, completion: nil)
```

Comment:
yes this was driving me crazy trying to keep this consistent. i think linting will complain if `completion` is not on the same line as the others, but other than that, this works for me.